media/js/firefox/new/all-test-variations.js

```diff
@@ -0,0 +1,132 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($, dataLayer) {
+    'use strict';
+
+    // check page version for tests
+    var version = Number($('#masthead').data('version'));
+
+    // adds sort value & (optional) new text to direct download li
+    var prepLi = function(li) {
+        var $li = $(li);
+        var $liA = $li.find('a');
+        var newOsName;
+
+        switch ($liA.text()) {
+        case 'Windows':
+            newOsName = 'Windows 32-bit';
+            $li.attr('data-sort', 1);
+            break;
+        case 'Linux':
+            newOsName = 'Linux 32-bit';
+            $li.attr('data-sort', 2);
+            break;
+        case 'Windows 64-bit':
+            $li.attr('data-sort', 3);
+            break;
+        case 'Linux 64-bit':
+            $li.attr('data-sort', 4);
+            break;
+        case 'OS X':
+            $li.attr('data-sort', 5);
+            break;
+        }
+
+        if (newOsName) {
+            $liA.text(newOsName);
+        }
+
+        return li;
+    };
+
+    var setupTest = function(version) {
+        var $dlButton = $('#download-button-desktop-release');
+        var $newLink;
+
+        // make sure desktop download button exists and user is on a recognized platform
+        if ($dlButton.length && $dlButton.find('.unrecognized-download:visible').length === 0) {
```

Comment:
Should we add a check here to make sure it's the desktop download button that's visible, and not the mobile button?